tools/esdlcmd/README.md

```diff
@@ -0,0 +1,424 @@
+
+Alternately, if you direct the ESP to read its dynamic configuration from files, then you'll need to create a *bundle*. This option is available when you run as an *esp application*. The esp application mode was created for cloud deployments but is also suitable for bare metal. Note that even when running in application mode, an ESP still requires a dali even if ESDL configurations aren't stored there. This dependency will be removed in a future release.
+
+### Manifest File
+
+A simplified example showing the format of a manifest file is below.
+
+```xml
+<em:Manifest xmlns:em="urn:hpcc:esdl:manifest">
+    <em:ServiceBinding service="WsFoobar" id="WsFoobar_desdl_binding">
+        <Methods>
+            <Scripts>
+                <em:Include file="WsFoobar-request-prep.xml"/>
+                <em:Include file="WsFoobar-logging-prep.xml"/>
+            </Scripts>
+            <Method name="FoobarSearch" url="127.0.0.1:8888">
+                <Scripts>
+                    <em:Include file="FoobarSearch-scripts.xml"/>
+                </Scripts>
+            </Method>
+        </Methods>
+        <LoggingManager>
+            <LogAgent transformSource="local" name="main-logging">
+                <LogDataXPath>
+                    <LogInfo name="PreparedData" xsl="log-prep">
+                </LogDataXPath>
+                <XSL>
+                    <Transform name="log-prep">
+                        <em:Include file="log-prep.xslt">
+                    </Transform>
+                </XSL>
+            </LogAgent>
+        </LoggingManager>
+    </em:ServiceBinding>
+    <em:ServiceDefinition>
+        <em:Include file="WsFoobar.ecm"/>
+    </em:ServiceDefinition>
+</em:Manifest>
+```
+
+The manifest nodes must use the namespace "urn:hpcc:esdl:manifest", and includes elements configuring asingle service.
+
+A `<em:ServiceBinding>` element contains the binding configuration for the service, including:
+
+1. Enumeration of all methods in the service.
+2. Configuration of the back-end of each method and any method-specific configuration.
+3. ESDL scripts defined in the scope of the service.
+4. ESDL scripts defined in the scope of each method.
+5. Logging configuration that may specify XSLT transforms a LogAgent needs.
+
+ServiceBinding Element Attributes
+
+Attribute   | Required? | Value  | Usage
+------------|-----------|--------|---------------
+`@id`       | yes       | string | Name of the binding to generate. Will appear in trace log output to identify this binding when loaded.
+`@service`  | yes       | string | Name of the service the generated binding will bind to. This service name must match the ESDL definition's service name.
+
+A `<em:ServiceDefinition>` element contains the ESDL definition of the service.
```

Comment:
Fixed typo. The tool doesn't prevent you from including multiples but the output would not be sensible. I think it would make sense to enforce only one per manifest file.